config/i18n-tasks.yml

```diff
@@ -94,6 +94,7 @@ ignore_unused:
 - 'errors.not_authorized'
 - 'experiments.*'
 - 'event_types.*'
+- 'headings.create_account_with*' # cannot find with interpolation
```

Comment:
Despite adding some duplication, I think a new template would make more sense and remove the need for the helper method. Ultimately, we're presenting different content, so this path will lead us to many `t('key.key.#{helper_method}` which is less revealing of purpose than just having separate templates.